packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -110,65 +110,26 @@ enum ListTileControlAffinity {
 /// )
 /// ```
 ///
-/// {@tool snippet}
+/// [ListTile] may need to be wrapped in a [Material] widget to animate
```

Comment:
> may need

Do they need to be wrapped individually in order for these animations to work? what other options are there?

Reply:
I place that line just below so if you're building the list_tile with Material, you can reference performance implications right there below those

https://github.com/flutter/flutter/pull/107104/files#diff-b399b26a1bee87ea31bd379fc20cabc50e9c70bcfe8e4dfd3c65c09ef2a5a80dR73

Reply:
Thank you for checking that! 

Can you clarify `may need`? If it is necessary in every case, it should just say `must`, if otherwise, when would you not need it? Is there another way?

Reply:
Thanks! That is one that folks always catch me on. 😜